testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -161,6 +162,45 @@ async fn wait_node_haproxy(
     .await
 }
 
+pub async fn check_for_container_restart(
+    kube_client: &K8sClient,
+    kube_namespace: &str,
+    sts_name: &str,
+) -> Result<()> {
+    aptos_retrier::retry_async(
```

Comment:
Been thinking of just using the `again` retrier

Reply:
Any benefit you see?

Reply:
Forge is the only piece of code that uses the `aptos_retrier`. It's a bit hard to read tbh

The `again` retrier is more straightforward and standard

Reply:
Sure, sounds good - let's do that as a follow-up as well. I can take care of it in subsequent PRs.